research/sentiment_analysis/sentiment_model.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+
+def _dynamic_pooling(w_embs):
```

Comment:
Why is custom pooling needed? And whatever the reason is, it should be documented.

Reply:
it is what the model suggests in the paper. Applying regular pooling with small kernel size reduces the performance. I will document it.